### PR TITLE
Feat/schedule

### PR DESCRIPTION
**Pull request by josesrodriguez610**

### CODE REVIEW

File: src/components/Settings/CardUserSettings.jsx [hunk 1]
```diff
@@ -25,11 +28,27 @@ function CardUserSettings() {
   const submitUser = () => {
     axios.put(`/user/user/${user?.id}`, { username })
       .then(() => axios.post('/user', { id: user?.id })
-        .then((response) => dispatch(setUser(response.data))));
+        .then((response) => dispatch(setUser(response.data))))
```
Comment (jstrzesz) [inline src/components/Settings/CardUserSettings.jsx]: tab back

File: src/components/Settings/CardUserSettings.jsx [hunk 2]
```diff
@@ -25,11 +28,27 @@ function CardUserSettings() {
...
+      .then(() => axios.get(`/league/league/${league}`)
+        .then((response) => dispatch(setUsersInLeague(response.data))))
```
Comment (jstrzesz) [inline src/components/Settings/CardUserSettings.jsx]: tab back

File: src/components/Settings/CardUserSettings.jsx [hunk 3]
```diff
@@ -25,11 +28,27 @@ function CardUserSettings() {
...
+      .catch((err) => console.warm(err));
     setUsername('');
   };
-  const submitTeam = () => setTeamName('');
-  const submitLogo = () => setTeamLogo('');
+  const submitTeam = () => {
+    axios.put('/user/updateUserTeamName', { userID: user.id, leagueID: league, teamName })
+      .then(() => axios.post('/user', { id: user?.id })
+        .then((response) => dispatch(setUser(response.data))))
```
Comment (jstrzesz) [inline src/components/Settings/CardUserSettings.jsx]: tab back

File: src/components/Settings/CardUserSettings.jsx [hunk 4]
```diff
@@ -25,11 +28,27 @@ function CardUserSettings() {
...
+      .catch((err) => console.warn(err));
+    setTeamName('');
+  };
+
+  const submitLogo = () => {
+    axios.put('/user/updateUserTeamLogo', { userID: user.id, leagueID: league, teamLogo })
+      .then(() => axios.post('/user', { id: user?.id })
+        .then((response) => dispatch(setUser(response.data))))
```
Comment (jstrzesz) [inline src/components/Settings/CardUserSettings.jsx]: tab back

File: src/components/Settings/SettingsLeague.jsx [hunk 5]
```diff
@@ -43,7 +44,8 @@ function SettingsLeague({ myLeague, setMyLeague }) {
         league_name: myLeague?.league_name,
         settings: leagueForm
       })
-      .then((response) => dispatch(setSettings(response.data)))
+      .then(() => axios.get(`/league/settings/${league}`)
+        .then((response) => dispatch(setSettings(response.data))))
```
Comment (jstrzesz) [inline src/components/Settings/SettingsLeague.jsx:48]: tab back

File: src/views/Schedule.jsx [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+        <div className='schedule'>
+          <Link key='home' className='nav_link' to='/'>
+            <h1>
+              {'A schedule hasn\'t been set at the moment or pick a league'}
+              {' '}
+            </h1>
+          </Link>
+        </div>
+      ) : (
+        <div className='schedule'>
```
Comment (jstrzesz) [inline src/views/Schedule.jsx:40]: extract to separate component

**Review by scottanderson1130 — APPROVED** [review]

**Review by ChristopherStumpe — APPROVED** [review]

**Pull request merged**